static/app/components/quickTrace/quickTrace.spec.tsx

```diff
@@ -142,13 +141,13 @@ describe('Quick Trace', function () {
           organization={organization}
         />
       );
-      const nodes = quickTrace.find('EventNode');
+      const nodes = await findAllByTestId('event-node');
       expect(nodes.length).toEqual(1);
-      expect(nodes.first().text()).toEqual('This Event');
+      expect(first(nodes)).toHaveTextContent('This Event');
```

Comment:
```suggestion
      expect(nodes[0]).toHaveTextContent('This Event');
```
then you don't need to import lodash  :)

Reply:
yeah, it's not needed.